_posts/2016-07-26-foreman-community-newsletter-july-2016.md

```diff
@@ -0,0 +1,132 @@
+not forgotten! It's been a busy month, and all the details are coming right up.
+
+### Foreman's 7th Birthday!
+
+On July 13th, Foreman turned 7 years old! Parties have been happening in some
+of our communities (see the [birthday events][birthday] page) and there's still
+a few to come.
+
+[birthday]: https://theforeman.org/2016/06/foremans-7th-birthday-events.html
+
+7 years is quite a milestone - and we're still growing! We'd like to thank
+everyone who's participated in the project over the years. Whether it's coding,
+testing, documenting, translating, reporting bugs... you're all amazing, and
+Foreman wouldn't be where it is today without the community that has developed
+around it. Thank you all!
+
+### 1.12 released
+
+The latest release of Foreman, 1.12.0, was released on 11th July, bringing with
+it the anticipated Puppet 4 support, and support for Ubuntu 16.04 (Xenial) in
+the package repos. That was followed by 1.12.1 on July 25th, fixed a lot of
+bugs as well as a couple of security issues. As ever, you can find out more
+about [1.12.0][12dot0] and [1.12.1][12dot1] on the announcements.
+
+[12dot0]: https://groups.google.com/forum/#!msg/foreman-announce/62FQvVQNZl8/B5SRjFKYBAAJ
+[12dot1]: https://groups.google.com/forum/#!topic/foreman-announce/XCVIT8lDJxI/discussion
+
+If you're upgrading from Puppet 3 to Puppet 4 as part of your 1.12 upgrade, you
+may want to take a look at the [migration page][p3to4] we've put together to
+document the upgrade steps. This has been confirmed working by a few people,
+but as with all complex upgrades, do take care.
+
+[p3to4]: http://projects.theforeman.org/projects/foreman/wiki/Upgrading_from_Puppet_3_to_4
+
+### 1.11.4 bugfix release
+
+The final release in our 1.11 series, 1.11.4 was released on July 25th. This
+brings 3 CVE fixes as well as numerous bug fixes. It's also likely to be the
+last release in the 1.11 series, so start planning your 1.12 upgrade. You can
+learn more from the [announcement][11dot4].
+
+[11dot4]: https://groups.google.com/forum/#!topic/foreman-announce/MmkL11GYUdM
+
+
+### Katello 3.0.2 released and 3.1 in RC phase
+
+The latest release in the Katello 3.0 line was released on July 6th, bringing
+bugfixes as well as resolutions for some significant upgrade issues. More
+details are available in the [announcement][30dot2]
+
+[30dot2]: https://groups.google.com/d/topic/foreman-users/8frDaBJObQ4/discussion
+
+Katello 3.1 is also in the Release Candidtae phase, and as ever, your testing
+is greatly valued. If you'd like to help with testing, you can get involved
+[here][31rc1]. Thanks to all our awesome testers!
+
+[31rc1]: https://groups.google.com/d/topic/foreman-users/ARgW31tXj7w/discussion
+
+### Foreman Ansible 1.0, now with support for roles
+
+Ansible is a big topic in the community right now, so it's great news that
+Daniel has released Foreman Ansible 1.0 with support for Ansible roles! He's
```

Comment:
Maybe have my name link to my github profile or say dlobatog,  I doubt readers will know who's exactly 'Daniel'